hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBStoreAbstractIterator.java

```diff
@@ -150,5 +181,21 @@ public final void removeFromDB() throws IOException {
   @Override
   public void close() {
     rocksDBIterator.close();
+    closed = true;
+    releaseEntry();
+  }
+
+  public static final class AutoCloseableRawKeyValue<RAW> extends RawKeyValue<RAW> implements AutoCloseable {
```

Comment:
Can rename class as CloseableRawKeyValue, can remove Auto keyword

Reply:
Closeable is for specifically throwing IOExceptions. AutoCloseable is the base interface. Since we are not throwing any exceptions we should call it AutoCloseableRawKeyValue ideally